react/src/components/Weather/CurrentWeather.js

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react';
+import CurrentStyle from './CurrentW.module.css';
+
+const api = {
+    key: 'fd54d9727b22aa867dc1f33db65eaafc',
```

Comment:
Rada na przyszłość, jak będziecie tworzyć następne projekty (nie ważne czy swoje czy dla jakiegoś klienta) tego typu dane lepiej wyciągać z zewnętrznego źródła (backend, plik properties etc)
Dlaczego? 
Historie gita można przeglądać, jeśli zdarzy wam sie źle skonfigurować repo / ktoś niepożądany sklonuje je sobie to będzie miał dostęp do waszych kluczy API a to może doprowadzić do wycieku waszych danych albo do znacznego zwiększenia opłat jeśli korzystacie z jakiegoś odpłatnego API.

Podejrzewam że tym kluczem wiele sie zrobić nie da więc o zmiany nie proszę, ale miejcie to na uwadze :)